### DISCUSSION TITLE
reference  tab

### DISCUSSION THREAD

**Discussion by mercabun** [keiwando/toonsquid#478] [Ideas]
when i'm drawing out my key poses, typically on an much higher quality animation software, i can place my character model sheet in reference tab so i can look at it while drawing on my canvas. i think this would be a bit helpful instead of uploading an image layer onto the canvas and having to turn it on and off multiple times while animating!
Though this could also be solved with allowing the pixel layer to be drawn outside of the canvas too  
![IMG_4655](https://github.com/keiwando/toonsquid/assets/133706192/8132caf3-ffc0-4f04-bc02-95651a709784)

**Comment by kevincreative** [comment]
Great idea! Maybe as a floating/zoomable window/panel?

**Reply by mercabun** [reply]
Yeah that works too!!
